README.md

```diff
@@ -74,6 +74,12 @@ Reporting task to send provenance events considered as errors by email.
 
 <img src="docs/images/email-reporting-task-configuration.png" width=600 />
 
+* `Email Subject Prefix` is a property that contains a prefix to be added in the email subject. It can be for example the nifi instance the error is coming from.
```

Comment:
```suggestion
* `Email Subject Prefix` is a property that contains a prefix to be added in the email subject. It can be for example the NiFi instance the error is coming from (optional).
```